openshift/README.md

```diff
@@ -0,0 +1,40 @@
+## Registry CI/CD pipline setup.
+
+Regsitry app has a full CI/CD pipline using GitHub Action and Argo CD. and its been hosted on **Silver** Cluster.
+
+**CI**: [Github Action ](https://github.com/bcgov/platform-services-registry/actions) will trigger image build accordingly and commit to the [tenant-gitops-platform-registry](https://github.com/bcgov-c/tenant-gitops-platform-registry/tree/main/platform-registry-app) based on the target deployment environment(the job triggered).
+
+**CD**: [ArgoCD resource](https://argocd-shared.apps.silver.devops.gov.bc.ca/applications?proj=&sync=&health=&namespace=&cluster=&labels=&search=platform-registry) is will compare the current deployment configration and the record we have in[tenant-gitops-platform-registry](https://github.com/bcgov-c/tenant-gitops-platform-registry/tree/main/platform-registry-app) and do the deploy the application accordingly.
+
+### Environment has different usage
```

Comment:
"Each Environment has a different use", or "Environments have different uses", sounds more natural to me as a native English speaker.